vsphere/internal/vmworkflow/virtual_machine_customize_subresource.go

```diff
@@ -453,6 +454,22 @@ func expandCustomizationIPSettings(d *schema.ResourceData, n int, v4gwAdd, v6gwA
 	return obj, v4gwFound, v6gwFound
 }
 
+func GetCustomIPFromSpec(d *schema.ResourceData, prefix string) string {
```

Comment:
Should add comments to describe what this function does.

Reply:
Done